site/src/components/AuthProvider/AuthProvider.tsx

```diff
@@ -0,0 +1,36 @@
+import { useActor, useInterpret } from "@xstate/react"
```

Comment:
I think we should move all Providers into a provider or state directory instead of them living mixed in with our components.

Reply:
Actually I would call it `contexts`. And I might rename these files to `AuthContext`, `DashboardContext`, etc.

Reply:
Totally